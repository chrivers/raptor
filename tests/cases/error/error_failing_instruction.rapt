# line 1
RUN /missing
# line 3
