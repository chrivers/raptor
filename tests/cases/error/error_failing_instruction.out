[1;34m[[0m[1;31mE[0m[1;34m][0m Error: No such file or directory (os error 2)
error: Error while executing instruction
 --> tests/cases/error/error_failing_instruction.rapt:1:1
  |
1 | RUN /missing
  | ^^^^^^^^^^^^ Sandbox request error: Sandbox error: ENOENT: No such file or directory
2 |
  |
